File: app/src/main/java/pl/expert/mobilewzr/data/WzrService.java
package pl.expert.mobilewzr.data;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

import java.util.List;

public interface WzrService {

    @GET("/.csv/plan_st.php?f2=4")
    Call<List<Subject>> listSubjects(@Query("f1") String groupId);
}

File: app/src/main/java/pl/expert/mobilewzr/data/WZRService.kt
package pl.expert.mobilewzr.data

import pl.expert.mobilewzr.data.model.Subject
import retrofit2.http.GET
import retrofit2.http.Query

interface WZRService {

    @GET(URLs.SUBJECTS_PATH)
    suspend fun getSubjects(
        @Query("f1") groupId: String,
        @Query("jp") params: String,
        @Query("f2") number: Int = 4
    ): List<Subject>

}